### PR TITLE
Added aria-label  tags for twitter image in home page

### PR DESCRIPTION
**Pull request by surya474**
Signed-off-by: surya474 <gantasuryateja474@gmail.com>

Related to [issue](https://github.com/containerd/containerd.io/issues/43)

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -64,7 +64,7 @@
         <hr class="hr has-background-grey">
 
         <div class="buttons">
-          <a class="button has-background-twitter-blue is-borderless has-text-white" href="{{ $twitterUrl }}">
+          <a class="button has-background-twitter-blue is-borderless has-text-white" aria-label="Know more in twitter" href="{{ $twitterUrl }}">
```
Comment (estesp) [inline layouts/partials/footer.html]: "Follow us on Twitter"

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -73,7 +73,7 @@
             </span>
           </a>
 
-          <a class="button is-borderless has-background-black has-text-white" href="{{ $githubUrl }}">
+          <a class="button is-borderless has-background-black has-text-white"  aria-label="Know more in Github" href="{{ $githubUrl }}">
```
Comment (estesp) [inline layouts/partials/footer.html]: "Visit our project on GitHub"

File: layouts/partials/navbar.html [hunk 3]
```diff
@@ -72,13 +72,13 @@
         <div class="navbar-item">
           <div class="field is-grouped">
             <p class="control">
-              <a class="button has-background-twitter-blue is-borderless has-text-white" href="{{ $twitterUrl }}">
+              <a class="button has-background-twitter-blue is-borderless has-text-white" aria-label="Know more in twitter" href="{{ $twitterUrl }}">
```
Comment (estesp) [inline layouts/partials/navbar.html]: "Follow us on Twitter"

File: layouts/partials/navbar.html [hunk 4]
```diff
@@ -72,13 +72,13 @@
...
                 <span class="icon">
                   <i class="fab fa-twitter" aria-hidden="true"></i>
                 </span>
               </a>
 
-              <a class="button is-borderless has-background-black has-text-white" href="{{ $githubUrl }}">
+              <a class="button is-borderless has-background-black has-text-white"  aria-label="Know more in Github" href="{{ $githubUrl }}">
```
Comment (estesp) [inline layouts/partials/navbar.html]: "Visit our project on GitHub"

**Comment by dmcgowan** [comment]
ping @surya474 

please update the wording

**Comment by thaJeztah** [comment]
ping @surya474 could you;

- update the PR, addressing the review comments
- squash the commits to a single commit
- update your DCO sign-off to use your real name (currently it's `Signed-off-by: surya474 <gantasuryateja474@gmail.com>`, please change `surya474` to your real name)

**Comment by dmcgowan** [comment]
Closing due to inactivity, feel free to re-open the pull request anytime if you continue working on this.

**Pull request closed without merging**